### PR TITLE
A0-2318: Update the no-extension warning

### PR DESCRIPTION
**Pull request by jalooc**
As discussed with Mateusz Raczyński and @Kocik.

### CODE REVIEW

File: packages/page-accounts/src/Accounts/BannerExtension.tsx [hunk 1]
```diff
@@ -53,10 +53,23 @@ function BannerExtension (): React.ReactElement | null {
...
-      <p>{phishing.current}</p>
+      <p>
+        {t<string>('Ensure that:')}
+        <SafetyInfoList>
+          <li>{t<string>('the extension has accounts,')}</li>
+          <li>{t<string>('at least one account is available for this chain,')}</li>
+          <li>{t<string>('the extension allows azero.dev to access accounts')}</li>
+        </SafetyInfoList>
+      </p>
+      <p>{t<string>('For extra protection, consider using the Threat Slayer extension which protects you from dangerous websites in real-time.')}</p>
```
Comment (youPickItUp) [inline packages/page-accounts/src/Accounts/BannerExtension.tsx]: Currently we only show this message if user has an extension and no accounts are injected, which is basically almost never. Perhaps we'd want to show this more frequently?

**Review by youPickItUp — APPROVED** [review]

File: packages/page-accounts/src/Accounts/BannerExtension.tsx (hunk 1 above)
Comment (jalooc) [reply packages/page-accounts/src/Accounts/BannerExtension.tsx]: I'll raise the question on slack.

Comment (jalooc) [reply packages/page-accounts/src/Accounts/BannerExtension.tsx]: Updated after a discussion with marketing:
image
image

Comment (jalooc) [reply packages/page-accounts/src/Accounts/BannerExtension.tsx]: Just noticed I had the one paragraph duplicated - fixed already.

**Review by Kocik — APPROVED** [review]

**Pull request merged**